draft-ietf-quic-v2.md

```diff
@@ -178,23 +178,22 @@ server MUST include the relevant transport parameters to validate that the
 server sent the Retry and the connection IDs used in the exchange, as described
 in {{Section 7.3 of QUIC}}.
 
-The server SHOULD start sending its Initial packets using the negotiated
-version as soon as it decides to change. Before the server is able to process
-transport parameters from the client, it might need to respond to Initial
-packets from the client. For these packets the server uses the original version.
+As the server cannot send CRYPTO frames until it has processed the client's
+transport parameters, it MUST send all CRYPTO frames in Initial packets using
+the negotiated version. The client learns the negotiated version from the first
```

Comment:
In practice, the client will learn before that because it needs to install initial keys in the negotiated version before it can access the CRYPTO frame. I'd suggest something like `The client learns the negotiated version by observing the first long header Version field that differs from the original version. If the client receives a CRYPTO frame from the server in the original version, that indicates that the negotiated version is equal to the original version.`